docs/user_guide.md

```diff
@@ -719,6 +721,200 @@ If you are a project admin, you can disable a Robot Account by clicking "Disable
+|Third|Tags to retain|Identify the tag or tags on which to apply the rule. You can identify tags that either have a certain name or name fragment, or that do not have that name or name fragment. Wild cards (for example `*tag`, `tag*`, and `**`) and regular expressions are permitted.|
+
+The repository filter is applied first to mark the repositories to which to apply the retention rule. 
+
+#### Example 1
+
+- You have 5 repositories in a project, repositories A to E.
+  - Repository A has 100 image tags, all of which have been pulled in the last week.
+  - Repositories B to E each have 6 images, none of which have been pulled in the last month.
+- You set the repository filter to `**`, meaning that all repositories in the project are included.
+- You set the retention policy to retain the 10 most recently pulled images in each repository.
+- You set the tag filter to `**`, meaning that all tags in the repository are included.
+
+In this example the rule retains the 10 most recently pulled images in repository A, and all of the images in repositories B to E, retaining a total of 34 image tags in the project. In other words, the rule does not treat all of the images in repositories A to E as a single pool from which to choose the 10 most recent images. So, even if the 11th to 100th tags in repository A have been pulled more recently than any of the tags in repositories B to E, all of the tags in repositories B to E are retained, because each of those repositories has fewer than 10 tags.
+
+#### Example 2
+
+This example uses the same project and repositories as example 1, but sets the retention policy to retain the images in each repository that have been pulled in the last 7 days.
+
+In this case, all of the images in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+
+### Combining Rules on a Respository
+
+You can define up to 15 rules per project. You can apply multiple rules to a repository or set of repositories. When you apply multiple rules to a repository, they are applied with `OR` logic rather than with `AND` logic. In this way, there is no prioritization of application of the rules on a given repository. 
+
+#### Example 3
+
+This example uses the same project and repositories as examples 1 and 2, but sets two rules:
+
+- Rule 1: Retain all of the images in each repository that have been pulled in the last 7 days.
+- Rule 2: Retain a maximum number of 10 images in each repository.
+
+In this case, repository A only retains the 10 most recent images, even though images 11 to 100 have been pulled in the last week. Repositories B to E retain no images, because none of those images have been pulled in the last week. In this example, 10 images are retained, as opposed to 34 images in example 1 and 100 in example 2.

```

Comment:
Not clear and seems something wrong here.

For repository A, rule 1 will retain all of the images as they have been pulled in the last week;
For repository A, rule 2 will retain 10 pulled images

In summary, with `OR` relationship, for repository A, all 100 images are retained.

For repository B-E, rule 1 will retain 0 images as no images are pulled in the last week;
For repository B-E, rule 2 will retain 6 (6<10) images

In summary, for repository B-E, each repository will keep 6 images there.

Reply:
I agree with @steven-zou .  @stuclem

Reply:
Updated to the following in https://github.com/goharbor/harbor/pull/8869/commits/f15d012a403726b6f59604e2418e11c0793ab5e1:

For repository A, rule 1 retains all of the images because they have all been pulled in the last week. Rule 2 retains the 10 most recently pulled images. So, since the two rules are applied with an `OR` relationship, all 100 images are retained in repository A.

For repositories B-E, rule 1 will retain 0 images as no images are pulled in the last week. Rule 2 will retain all 6 images because 6 < 10. So, since the two rules are applied with an `OR` relationship, for repositories B-E, each repository will keep all 6 images.

In this example, all of the images are retained.

Reply:
lgtm